sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -56,6 +57,57 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return this.repo.save(saveLocation);
   }
 
+  /**
+   * Updating institution location.
+   * @param data Payload of updated data.
+   * @param locationId Location Id to update.
+   * @returns Updated location as InstitutionLocation.
+   */
+  async updateLocationPrimaryContact(
+    data: InstitutionLocationPrimaryContactModel,
+    locationId: number,
+  ): Promise<InstitutionLocation> {
+    const saveLocation = {
+      primaryContact: {
+        firstName: data.primaryContactFirstName,
+        lastName: data.primaryContactLastName,
+        email: data.primaryContactEmail,
+        phone: data.primaryContactPhone,
+      },
+      id: locationId,
+    };
+
+    return this.repo.save(saveLocation);
+  }
+
+  /**
+   * Updating institution location.
+   * @param data Payload of updated data.
+   * @param locationId Location Id to update.
+   * @returns Updated location as InstitutionLocation.
+   */
+  async updateLocationForAEST(
+    data: InstitutionLocationModel,
```

Comment:
Please rename this parameter. `data` is used as a "main" data and it is also a property being updated in the location.
![image](https://user-images.githubusercontent.com/61259237/167963143-75521f2c-6e91-480d-90ba-55e777fc5fb2.png)